src/test/resources/pages/instructorFeedbackEmptyAll.html

```diff
@@ -1096,7 +1096,256 @@ <h4 class="modal-title">
 </script>
 <script src="/js/instructorFeedbacks.js" type="text/javascript">
 </script>
-<div class="ui-datepicker ui-widget ui-widget-content ui-helper-clearfix ui-corner-all" id="ui-datepicker-div">
+<div class="ui-datepicker ui-widget ui-widget-content ui-helper-clearfix ui-corner-all" id="ui-datepicker-div" style="display: none;">
```

Comment:
Looks like this has become unpredictable HTML as its content depends on the current date. Use this patch to detect and ignore it:

```patch
---
 .../java/teammates/test/driver/HtmlHelper.java     |  14 ++
 src/test/resources/pages/godmode.html              |   3 +
 .../resources/pages/godmodeExpectedOutput.html     |   1 +
 .../resources/pages/godmodeExpectedPartOutput.html |   1 +
 .../pages/instructorFeedbackEmptyAll.html          | 252 +--------------------
 5 files changed, 20 insertions(+), 251 deletions(-)

diff --git a/src/test/java/teammates/test/driver/HtmlHelper.java b/src/test/java/teammates/test/driver/HtmlHelper.java
index 7589225..a3982d0 100644
--- a/src/test/java/teammates/test/driver/HtmlHelper.java
+++ b/src/test/java/teammates/test/driver/HtmlHelper.java
@@ -162,6 +162,9 @@ public final class HtmlHelper {
                 } else if (isMotdContainerAttribute(attribute)) {
                     // replace MOTD content with placeholder
                     return generateStudentMotdPlaceholder(indentation);
+                } else if (isDatepickerAttribute(attribute)) {
+                    // replace datepicker with placeholder
+                    return generateDatepickerPlaceholder(indentation);
                 }
             }
         } else if (currentNode.getNodeName().equalsIgnoreCase("select")) {
@@ -199,6 +202,10 @@ public final class HtmlHelper {
         return indentation + "${timezone.options}" + System.lineSeparator();
     }
 
+    private static String generateDatepickerPlaceholder(String indentation) {
+        return indentation + "${datepicker}" + System.lineSeparator();
+    }
+
     // private static String generateTinymceStylePlaceholder(String indentation) {
     //     return indentation + "${tinymce.style}" + System.lineSeparator();
     // }
@@ -280,6 +287,13 @@ public final class HtmlHelper {
     }
 
     /**
+     * Checks for datepicker (i.e a <code>div</code> with id <code>ui-datepicker-div</code>).
+     */
+    private static boolean isDatepickerAttribute(Node attribute) {
+        return checkForAttributeWithSpecificValue(attribute, "id", "ui-datepicker-div");
+    }
+
+    /**
      * Checks for timezone selectors (i.e a <code>select</code> with id <code>coursetimezone</code>).
      */
     private static boolean isTimeZoneSelectorAttribute(Node attribute) {
diff --git a/src/test/resources/pages/godmode.html b/src/test/resources/pages/godmode.html
index e3a3125..ecae76a 100644
--- a/src/test/resources/pages/godmode.html
+++ b/src/test/resources/pages/godmode.html
@@ -44,6 +44,9 @@
                 <li><!-- now.datetime.sessions --></li>
                 <li><!-- now.datetime.iso8601utc --></li>
                 <li><!-- now.datetime.courses --></li>
+                <li><div class="ui-datepicker ui-widget ui-widget-content ui-helper-clearfix ui-corner-all" id="ui-datepicker-div" style="display: none;">
+                    <div class="ui-datepicker-header ui-widget-header ui-helper-clearfix ui-corner-all"></div>
+                </div></li>
                 <li><div class="col-md-8" id="adminInstitute">
                 </div></li>
                 <li><div id="adminInstitute" class="col-md-8">
diff --git a/src/test/resources/pages/godmodeExpectedOutput.html b/src/test/resources/pages/godmodeExpectedOutput.html
index 9da3bb5..e1700f1 100644
--- a/src/test/resources/pages/godmodeExpectedOutput.html
+++ b/src/test/resources/pages/godmodeExpectedOutput.html
@@ -41,6 +41,7 @@
                 <li>${datetime.now} UTC+0000</li>
                 <li>${datetime.now.iso8601utc}</li>
                 <li>${datetime.now.courses}</li>
+                <li>${datepicker}</li>
                 <li>${admin.institute}</li>
                 <li>${admin.institute}</li>
                 <li><input name="token" type="hidden" value="${sessionToken}"></li>
diff --git a/src/test/resources/pages/godmodeExpectedPartOutput.html b/src/test/resources/pages/godmodeExpectedPartOutput.html
index 0e99a22..ed97e5a 100644
--- a/src/test/resources/pages/godmodeExpectedPartOutput.html
+++ b/src/test/resources/pages/godmodeExpectedPartOutput.html
@@ -39,6 +39,7 @@
         <li>${datetime.now} UTC+0000</li>
         <li>${datetime.now.iso8601utc}</li>
         <li>${datetime.now.courses}</li>
+        <li>${datepicker}</li>
         <li>${admin.institute}</li>
         <li>${admin.institute}</li>
         <li><input name="token" type="hidden" value="${sessionToken}"></li>
diff --git a/src/test/resources/pages/instructorFeedbackEmptyAll.html b/src/test/resources/pages/instructorFeedbackEmptyAll.html
index 291a02d..6084663 100644
--- a/src/test/resources/pages/instructorFeedbackEmptyAll.html
+++ b/src/test/resources/pages/instructorFeedbackEmptyAll.html
@@ -1096,256 +1096,6 @@
 </script>
 <script src="/js/instructorFeedbacks.js" type="text/javascript">
 </script>
-<div class="ui-datepicker ui-widget ui-widget-content ui-helper-clearfix ui-corner-all" id="ui-datepicker-div" style="display: none;">
-  <div class="ui-datepicker-header ui-widget-header ui-helper-clearfix ui-corner-all">
-    <a class="ui-datepicker-prev ui-corner-all" data-event="click" data-handler="prev" title="Prev">
-      <span class="ui-icon ui-icon-circle-triangle-w">
-        Prev
-      </span>
-    </a>
-    <a class="ui-datepicker-next ui-corner-all" data-event="click" data-handler="next" title="Next">
-      <span class="ui-icon ui-icon-circle-triangle-e">
-        Next
-      </span>
-    </a>
-    <div class="ui-datepicker-title">
-      <span class="ui-datepicker-month">
-        March
-      </span>
-       
-      <span class="ui-datepicker-year">
-        2018
-      </span>
-    </div>
-  </div>
-  <table class="ui-datepicker-calendar">
-    <thead>
-      <tr>
-        <th class="ui-datepicker-week-end" scope="col">
-          <span title="Sunday">
-            Su
-          </span>
-        </th>
-        <th scope="col">
-          <span title="Monday">
-            Mo
-          </span>
-        </th>
-        <th scope="col">
-          <span title="Tuesday">
-            Tu
-          </span>
-        </th>
-        <th scope="col">
-          <span title="Wednesday">
-            We
-          </span>
-        </th>
-        <th scope="col">
-          <span title="Thursday">
-            Th
-          </span>
-        </th>
-        <th scope="col">
-          <span title="Friday">
-            Fr
-          </span>
-        </th>
-        <th class="ui-datepicker-week-end" scope="col">
-          <span title="Saturday">
-            Sa
-          </span>
-        </th>
-      </tr>
-    </thead>
-    <tbody>
-      <tr>
-        <td class=" ui-datepicker-week-end ui-datepicker-other-month ui-datepicker-unselectable ui-state-disabled ">
-          <span class="ui-state-default">
-            25
-          </span>
-        </td>
-        <td class=" ui-datepicker-other-month ui-datepicker-unselectable ui-state-disabled ">
-          <span class="ui-state-default">
-            26
-          </span>
-        </td>
-        <td class=" ui-datepicker-other-month ui-datepicker-unselectable ui-state-disabled ">
-          <span class="ui-state-default">
-            27
-          </span>
-        </td>
-        <td class=" ui-datepicker-other-month ui-datepicker-unselectable ui-state-disabled ">
-          <span class="ui-state-default">
-            28
-          </span>
-        </td>
-        <td class=" " data-event="click" data-handler="selectDay" data-month="2" data-year="2018">
-          <a class="ui-state-default" href="#">
-            1
-          </a>
-        </td>
-        <td class=" ui-datepicker-days-cell-over  ui-datepicker-current-day ui-datepicker-today" data-event="click" data-handler="selectDay" data-month="2" data-year="2018">
-          <a class="ui-state-default ui-state-highlight ui-state-active ui-state-hover" href="#">
-            2
-          </a>
-        </td>
-        <td class=" ui-datepicker-week-end " data-event="click" data-handler="selectDay" data-month="2" data-year="2018">
-          <a class="ui-state-default" href="#">
-            3
-          </a>
-        </td>
-      </tr>
-      <tr>
-        <td class=" ui-datepicker-week-end " data-event="click" data-handler="selectDay" data-month="2" data-year="2018">
-          <a class="ui-state-default" href="#">
-            4
-          </a>
-        </td>
-        <td class=" " data-event="click" data-handler="selectDay" data-month="2" data-year="2018">
-          <a class="ui-state-default" href="#">
-            5
-          </a>
-        </td>
-        <td class=" " data-event="click" data-handler="selectDay" data-month="2" data-year="2018">
-          <a class="ui-state-default" href="#">
-            6
-          </a>
-        </td>
-        <td class=" " data-event="click" data-handler="selectDay" data-month="2" data-year="2018">
-          <a class="ui-state-default" href="#">
-            7
-          </a>
-        </td>
-        <td class=" " data-event="click" data-handler="selectDay" data-month="2" data-year="2018">
-          <a class="ui-state-default" href="#">
-            8
-          </a>
-        </td>
-        <td class=" " data-event="click" data-handler="selectDay" data-month="2" data-year="2018">
-          <a class="ui-state-default" href="#">
-            9
-          </a>
-        </td>
-        <td class=" ui-datepicker-week-end " data-event="click" data-handler="selectDay" data-month="2" data-year="2018">
-          <a class="ui-state-default" href="#">
-            10
-          </a>
-        </td>
-      </tr>
-      <tr>
-        <td class=" ui-datepicker-week-end " data-event="click" data-handler="selectDay" data-month="2" data-year="2018">
-          <a class="ui-state-default" href="#">
-            11
-          </a>
-        </td>
-        <td class=" " data-event="click" data-handler="selectDay" data-month="2" data-year="2018">
-          <a class="ui-state-default" href="#">
-            12
-          </a>
-        </td>
-        <td class=" " data-event="click" data-handler="selectDay" data-month="2" data-year="2018">
-          <a class="ui-state-default" href="#">
-            13
-          </a>
-        </td>
-        <td class=" " data-event="click" data-handler="selectDay" data-month="2" data-year="2018">
-          <a class="ui-state-default" href="#">
-            14
-          </a>
-        </td>
-        <td class=" " data-event="click" data-handler="selectDay" data-month="2" data-year="2018">
-          <a class="ui-state-default" href="#">
-            15
-          </a>
-        </td>
-        <td class=" " data-event="click" data-handler="selectDay" data-month="2" data-year="2018">
-          <a class="ui-state-default" href="#">
-            16
-          </a>
-        </td>
-        <td class=" ui-datepicker-week-end " data-event="click" data-handler="selectDay" data-month="2" data-year="2018">
-          <a class="ui-state-default" href="#">
-            17
-          </a>
-        </td>
-      </tr>
-      <tr>
-        <td class=" ui-datepicker-week-end " data-event="click" data-handler="selectDay" data-month="2" data-year="2018">
-          <a class="ui-state-default" href="#">
-            18
-          </a>
-        </td>
-        <td class=" " data-event="click" data-handler="selectDay" data-month="2" data-year="2018">
-          <a class="ui-state-default" href="#">
-            19
-          </a>
-        </td>
-        <td class=" " data-event="click" data-handler="selectDay" data-month="2" data-year="2018">
-          <a class="ui-state-default" href="#">
-            20
-          </a>
-        </td>
-        <td class=" " data-event="click" data-handler="selectDay" data-month="2" data-year="2018">
-          <a class="ui-state-default" href="#">
-            21
-          </a>
-        </td>
-        <td class=" " data-event="click" data-handler="selectDay" data-month="2" data-year="2018">
-          <a class="ui-state-default" href="#">
-            22
-          </a>
-        </td>
-        <td class=" " data-event="click" data-handler="selectDay" data-month="2" data-year="2018">
-          <a class="ui-state-default" href="#">
-            23
-          </a>
-        </td>
-        <td class=" ui-datepicker-week-end " data-event="click" data-handler="selectDay" data-month="2" data-year="2018">
-          <a class="ui-state-default" href="#">
-            24
-          </a>
-        </td>
-      </tr>
-      <tr>
-        <td class=" ui-datepicker-week-end " data-event="click" data-handler="selectDay" data-month="2" data-year="2018">
-          <a class="ui-state-default" href="#">
-            25
-          </a>
-        </td>
-        <td class=" " data-event="click" data-handler="selectDay" data-month="2" data-year="2018">
-          <a class="ui-state-default" href="#">
-            26
-          </a>
-        </td>
-        <td class=" " data-event="click" data-handler="selectDay" data-month="2" data-year="2018">
-          <a class="ui-state-default" href="#">
-            27
-          </a>
-        </td>
-        <td class=" " data-event="click" data-handler="selectDay" data-month="2" data-year="2018">
-          <a class="ui-state-default" href="#">
-            28
-          </a>
-        </td>
-        <td class=" " data-event="click" data-handler="selectDay" data-month="2" data-year="2018">
-          <a class="ui-state-default" href="#">
-            29
-          </a>
-        </td>
-        <td class=" " data-event="click" data-handler="selectDay" data-month="2" data-year="2018">
-          <a class="ui-state-default" href="#">
-            30
-          </a>
-        </td>
-        <td class=" ui-datepicker-week-end " data-event="click" data-handler="selectDay" data-month="2" data-year="2018">
-          <a class="ui-state-default" href="#">
-            31
-          </a>
-        </td>
-      </tr>
-    </tbody>
-  </table>
-</div>
+${datepicker}
 </body>
 </html>
-- 
```